src/offline/DefaultConfiguration.js

```diff
@@ -2,17 +2,31 @@ goog.module('ngeo.offline.DefaultConfiguration');
 goog.module.declareLegacyNamespace();
 
 goog.require('ol.Observable');
+goog.require('ol.layer.Layer');
+goog.require('ol.layer.Vector');
+goog.require('ol.layer.Tile');
+goog.require('ol.layer.Image');
+
 goog.require('ngeo.CustomEvent');
 
 const utils = goog.require('ngeo.offline.utils');
 
+
 /**
  * @implements {ngeox.OfflineConfiguration}
  */
 exports = class extends ol.Observable {
 
-  constructor() {
+  /**
+   * @param {!angular.Scope} $rootScope The rootScope provider.
+   */
+  constructor($rootScope) {
     super();
+    localforage.config({
+      'name': 'ngeoOfflineStorage',
+      'version': 1.0,
+      'storeName': 'storage'
```

Comment:
It is a problem if later, we have another storage in ngeo, and we change the name here ? (will the users loose their storage ?). If we have any doubt, we should call it "offlineStorage" or similar.